practice-one/src/index.html

```diff
@@ -38,13 +35,48 @@
         <a class="btn btn-link btn-help-center" href="#">
           Help Center
         </a>
-
       </nav>
-
     </div>
-
   </header>
+  <!-- END HEADER -->
+
+  <!-- START MAIN -->
+  <main class="main-content">
+    <div class="container">
+      <!-- START SECTION GET-STARTED -->
+      <section class="section-get-started">
+        <section class="download">
+          <h1 class="download-heading">
+            Download app,
+            save money, make
+            friends!
+          </h1>
+          <p class="content download-content">
+            Its simple and its free. Play your part in reducing Carbon Footprint and help
+            Mother Nature to sustain its beauty. So what are you waiting for? Lets ride
+            together
+          </p>
 
+          <div class="download-links">
+            <a href="#" class="btn btn-link btn-download">
+              <img src="./assets/images/download-icon-store.svg" alt="download-icon-store">
```

Comment:
The layout will break when the image doesn't load. So you should set the width and height for the image.

But I recommend you use a background image for this case.

```
<a>
  <i class="icon icon-app-store"></i>Download
</a>

.icon-app-store {
  background-image: .....
}
```